packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -365,14 +365,17 @@ export function useMulti<
       const newQuery = {...locationQuery, [queryLimitName]: newLimit}
       history.push({...location, search: `?${qs.stringify(newQuery)}`})
     }
+
+    const variables = {
+      ...graphQLVariables,
+      input: {
+        ...graphQLVariables.input,
+        terms: {...graphQLVariables.input.terms, limit: newLimit}
+      }
+    };
+
     void fetchMore({
-      variables: {
-        ...graphQLVariables,
-        input: {
-          ...graphQLVariables.input,
-          terms: {...graphQLVariables.input.terms, limit: newLimit}
-        }
-      },
+      variables,
```

Comment:
Typescript was giving me a similarly nonsensical error message about two types with the same name being otherwise unrelated here.  I suspect there's some issue with generic type variance and tsc's inference on spread interacting in weird ways.